### ISSUE TITLE
Get rid of charset WOM area and make it a ROM mirror

### ISSUE THREAD

**Issue by lydon42** [MEGA65/mega65-core#676]
Currently the freezer has no way of reading the current charset, and as such the charset can not be frozen.

@gardners suggested that I ask him to replace the WOM charset region by a mirror of the ROM charset instead...

I am oblivious to the consequences of that :)

@gardners adds: make mirror area ignore write protect on the ROM, so that the charset can be modified.

**Comment by dansanderson** [comment]
Please also consider this alternate proposal I filed a month ago: https://github.com/MEGA65/mega65-core/issues/667

**Comment by lydon42** [comment]
Test looks good, works well!

**Comment by dansanderson** [comment]
Can you summarize the accepted change? The issue title implies that writing to what used to be the char WOM area now overwrites the CHARC region of chip RAM (with write-only protection ignored). If that's true, that would have some significant backwards incompatible consequences.

If we went a different way, I'd like to understand it so I can update the Memory chapter.

**Comment by lgblgblgb** [comment]
@dansanderson @gardners I'm interested too ... As far as I know, the situation: the "char ROM" inside the "usual" ROM (which would have been used by C65) does (did?) not do anything at all. The real thing was the "WOM" (Write-Only-Memory). As far as I guessed, the change only is making the WOM readable (but still no connection between the "WOM" and the 128K's ROM content, and no change there), thus shouldn't be called WOM anymore :) So the "main ROM" is not affected at all. But this is only a guess of mine.

**Comment by lydon42** [comment]
The change is that the FREEZER can read the WOM, making freezing it possible.
